src/config/types.ts

```diff
@@ -15,18 +15,30 @@ export interface Provider<T> {
 }
 
 // App
-export type AppModeNames = 'VxMark' | 'VxPrint' | 'VxMark + VxPrint'
-export interface AppMode {
-  readonly name: AppModeNames
-  readonly isVxPrint?: boolean
-  readonly isVxMark?: boolean
-}
-export const VxPrintOnly: AppMode = { name: 'VxPrint', isVxPrint: true }
-export const VxMarkOnly: AppMode = { name: 'VxMark', isVxMark: true }
-export const VxMarkPlusVxPrint: AppMode = {
+export const VxPrintOnly = {
+  name: 'VxPrint',
+  isVxMark: false,
+  isVxPrint: true,
+} as const
+export const VxMarkOnly = {
+  name: 'VxMark',
+  isVxMark: true,
+  isVxPrint: false,
+} as const
+export const VxMarkPlusVxPrint = {
   name: 'VxMark + VxPrint',
   isVxPrint: true,
   isVxMark: true,
+} as const
+export type AppMode =
+  | typeof VxMarkOnly
```

Comment:
Q: should this line start with a pipe?

Reply:
That's what prettier did. It's just a way to visually make it easier to grok.

Reply:
Cool. Good to know this syntax is acceptable. I like.